part-08.mdx

```diff
@@ -0,0 +1,230 @@
+
+Keep in mind more javascript modules may be loaded (every chart library is a different javascript file, that is loaded on first use).
+
+## Set the default Netdata server
+
+`dashboard.js` will attempt to auto-detect the URL of the Netdata server it is loaded from, 
+and set this server as the default Netdata server for all charts.
+
+If you need to set any other URL as the default Netdata server for all charts that do not specify a Netdata server, 
+add this before loading `dashboard.js`:
+
+```js
+<script type="text/javascript">var netdataServer = "http://your.netdata.server:19999";</script>
+```
+
+## Create a chart, set its duration and size
+
+A chart is an individual, interactive, always-updating graphic displaying one or more collected/calculated metrics. 
+Collectors generate charts.
+
+Netdata displays a chart’s name in parentheses above the chart.
+
+<img width="1482" alt="An Example of Netdata Chart" src="https://user-images.githubusercontent.com/12263278/65382643-ffedef80-dd01-11e9-845a-a11fc7d4ba78.png">
+
+To add charts, you need to add a `div` for each of them. Each of these `div` elements accepts a few `data-` attributes:
+
+### The chart unique ID
+
+The unique ID of a chart is shown at the title of the chart of the default Netdata dashboard. 
+You can find all the charts and their data available at your Netdata server with this URL: `http://SERVER_IP:19999/api/v1/charts`.
+
+To specify the unique id, use this:
+
+```html
+<div data-netdata="unique.id"></div>
+```
+
+### The duration of the chart
+
+The duration of the chart is how much time of data it will show. 
+
+You can specify it using:
+
+```html
+<div data-netdata="unique.id"
+ data-after="AFTER_SECONDS"
+ data-before="BEFORE_SECONDS"
+ ></div>
+ ```
+`AFTER_SECONDS` and `BEFORE_SECONDS` are numbers representing a time-frame in seconds.
+
+These can be either:
+
+-   **absolute** Unix timestamps (in javascript terms, they are `new Date().getTime() / 1000`. 
+Using absolute timestamps, you can have a chart always showing the same time-frame.
+
+-   **relative** number of seconds to now. To show the last 10 minutes of data, 
+`AFTER_SECONDS` must be `-600` (relative to now), and `BEFORE_SECONDS` must be `0` (meaning: now).
+If you want the chart to auto-refresh the current values, you need to specify `relative` values.
+
+### Chart sizes
+
+You can set the size of the chart using this:
+
+```html
+<div data-netdata="unique.id"
+ data-width="WIDTH"
+ data-height="HEIGHT"
+ ></div>
+```
+
+`WIDTH` and `HEIGHT` can be anything CSS accepts for width and height (e.g. percentages, pixels, etc). 
+Keep in mind that for certain chart libraries, dashboard.js may apply an aspect ratio to these.
+
+If you want `dashboard.js` to permanently remember (browser local storage) the dimensions of the chart 
+(the user may resize it), you can add `data-id="SETTINGS_ID"`, 
+where `SETTINGS_ID` is anything that will be common for this chart across user sessions.
```

Comment:
What does "anything" mean? I feel like I still don't understand what I would put in for `SETTINGS_ID`.